Formula/quast.rb

```diff
@@ -14,20 +15,20 @@ class Quast < Formula
     sha256 "ce3487e38818178541943c3961139339cbfb6f785ffb744a0eddef8d28aabd90" => :x86_64_linux
   end
 
-  depends_on "matplotlib"
-
-  stable do
-    depends_on "e-mem"
-  end
+  depends_on "zlib" unless OS.mac?
+  depends_on "python"
 
   def install
-    # Remove the precompiled E-MEM binary.
-    rm_f "quast_libs/MUMmer/e-mem-osx"
     prefix.install Dir["*"]
-    bin.install_symlink "../quast.py", "../metaquast.py", "../icarus.py",
-      "quast.py" => "quast", "metaquast.py" => "metaquast", "icarus.py" => "icarus"
+    bin.install_symlink "../quast.py", "../metaquast.py", "../quast-lg.py", "../icarus.py",
+      "quast.py" => "quast", "metaquast.py" => "metaquast", "quast-lg.py" => "quast-lg", "icarus.py" => "icarus"
     # Compile the bundled aligner so that `brew test quast` does not fail.
     system "#{bin}/quast", "--test"
+    if OS.linux?
```

Comment:
Please move the `inreplace` above `system "#{bin}/quast", "--test"` and add a blank line before and after.

Reply:
Yes, sure. Sorry, I am testing `brew install --build-from-source` on Mac, so it is difficult to catch everything.

Reply:
No worries at all, Alexei.